pom.xml

```diff
@@ -121,6 +121,12 @@
             <version>1.9.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-library</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This is a duplicate and most likely redundant dependency at the topmost level.

Reply:
If you want _hamcrest-library_ to be available to all modules for unit tests, this is the right location to specify the dependency (along with junit itself). _hamcrest-core_ should also be moved here then.
The _dependencyManagement_ section is good when we want to specify a dependency selectively in a subset of modules but manage the attributes of that dependency (version, scope) in one place.
If _hamcrest_ dependencies are specified as global (spliceengine-parent dependencies) there is no need to add them to the _dependencyManagement_ section.

Reply:
I removed the hamcrest reference in _dependencyManagement_ and added hamcrest (without -core or -library) along with junit.